src/components/tabController/TabBar.tsx

```diff
@@ -317,19 +317,19 @@ const styles = StyleSheet.create({
     left: 0,
     width: 70,
     height: 2,
-    backgroundColor: Colors.primary
+    backgroundColor: Colors.$backgroundPrimary
   },
   containerShadow: {
     ...Platform.select({
       ios: {
-        shadowColor: Colors.grey10,
+        shadowColor: Colors.black,
```

Comment:
Why black?

Reply:
Private has this as `black`, and it removed the need for a new token, WDYT?

Reply:
But this way it'll be `black` in all modes.
Maybe `$backgroundNeutralHeavy` will do the work here since it's dark in light mode, and light in dark mode..

Reply:
We still don't have a solution for shadows AFAIK, so if we're not handling this in private, why should this be different?

Reply:
Ohh so you just align the color to the private one?

Reply:
Not really align everything, but this specifically because otherwise it's a new token for this edge case...

Reply:
I see, ok..